projects/instant-ui-library/src/lib/button/base-button/base-button.component.html

```diff
@@ -0,0 +1,17 @@
+<button
+  [ngClass]="[
+    'btn',
+    modeType === mode.PRIMARY ? 'btn--primary' : '',
+    modeType === mode.SECONDARY ? 'btn--secondary' : '',
+    modeType === mode.TERTIARY ? 'btn--tertiary' : '',
+    modeType === mode.HIGHLIGHT ? 'btn--highlight' : ''
+  ]"
+  [attr.aria-disabled]="disabled ? true : null"
```

Comment:
**Suggestion:** Thinking we might want something more for disabled state. Add better context somehow. Add icon to button? We should create a ticket for this.

Reply:
[Discuss disabled state](https://trello.com/c/7CQa42ll/21-discuss-disabled-states)